chef/cookbooks/ceph/recipes/radosgw_civetweb.rb

```diff
@@ -0,0 +1,78 @@
+# This recipe doesn't do anything other than deal with SSL keys and certificates.
+# Everything else for civetweb setup is in conf.rb.  It could arguably be merged
+# into radosgw.rb; it only exists as a separate file because it used to be
+# radosgw_apache2.rb
+
+if node["ceph"]["radosgw"]["ssl"]["enabled"]
```

Comment:
If nobody objects, I'd rather just leave it as is for the moment and live with the Hound warning, to save me having to test it again ;-)